api/preprints/serializers.py

```diff
@@ -362,36 +362,42 @@ def update(self, preprint, validated_data):
         if 'has_coi' in validated_data:
             try:
                 preprint.update_has_coi(auth, validated_data['has_coi'])
+                save_preprint = True
```

Comment:
As you'll see in my comment of your preprint model code, I think this might not be ideal, because you'll throw away information from an API request silently in the case of malformed api requests. If someone says that they don't have a coi, but puts something like "No conflict of interest" in the `conflict_of_interest_statement` field, then the COI Statement field is going to be thrown away and the API consumer won't realize it or know why. Better instead to modify the `update_*` methods to check for falsey versions of the information and to accept and save those rather than erroring, but still erroring when there's truthy information inappropriately added to the fields.